src/App.scss

```diff
@@ -39,3 +40,12 @@
     }
   }
 }
+
+.mapboxgl-ctrl-top-right {
+  top: auto!important;
+  bottom: 80px
+}
+
+#ucfhb {
+  position: fixed !important;
+}
```

Comment:
I think I see what you're trying to do with the `.mapboxgl-ctrl-top-right`. This is not necessary. Instead, and those components, change the position to `bottom-right` and they will position correctly down in the bottom right.